source/administration/object-management.rst

```diff
@@ -220,6 +220,43 @@ Conversion to or from XML may be required for importing rules created on S3 or s
 
 See :ref:`minio-lifecycle-management` for more complete documentation.
 
+Target Bucket Considerations
+----------------------------
+
+- MinIO does *not* require that the target bucket match object management or versioning configurations with the source bucket.
+- Object locking set on the target bucket may prevent desired ``delete`` operations from the source bucket from completing.
+- If the target bucket has expiration rules configured, these may result in data loss.
+- MinIO tiers objects with their own ``UUID``, so versioning on the target bucket is not required.
+
+  Versioning on the target bucket may have the following unexpected or undesired results:
```

Comment:
```suggestion
Versioning on the target bucket may have the following unexpected or undesired results:
```

Reply:
Or crunch this section down into

- Versioning on the target bucket may result in reduced storage efficiency due to the creation of ``DeleteMarkers`` in response to delete operations on the source

Reply:
Taking the second suggestion and simplifying it down to just that.